api/uuid-functions/uuid_timestamp_micros.md

```diff
@@ -0,0 +1,37 @@
+---
+api_name: uuid_timestamp_micros()
+excerpt: Extract a PostgreSQL timestamp with microsecond precision from a version 7 UUID
+topics: [uuid]
+keywords: [uuid]
+tags: [uuid, partitioning, events]
+api:
+  license: community
+  type: function
+products: [cloud, mst, self_hosted]
+---
+
+# uuid_timestamp_micros() <Tag type="Community">Community</Tag>
+
+Extract a PostgreSQL timestamp with time zone from a version 7
+UUID. The UUID contains a millisecond unix timestamp and an optional
+sub-millisecond fraction. Unlike `uuid_timestamp`, the microsecond
```

Comment:
Should probably add link to other function here.

Reply:
Done